science/iqtree/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               cmake 1.1
+
+name                    iqtree
+github.setup            Cibiv IQ-TREE 1.6.12
```

Comment:
this should become `github.setup            Cibiv IQ-TREE 1.6.12 v`

Reply:
Fixed.